tests/profiling/csvcubed_inspect_process.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from csvcubeddevtools.helpers.file import get_test_cases_dir
+from csvcubed.cli.build import build
+from memory_profiler import profile
+
+from tests.stress.buildpreprocess import generate_maximally_complex_csv
+
+# import the generate_maximaly_complex_scvfile function the create a csv file after that wun the build command in a temp directory(feed that directory to the inpect command bellow) then the inspect command will function.
+
+
+@profile(precision=4)
```

Comment:
Want to get rid of the `precision=4` bit since it didn't yield much use to us?

Reply:
done